tock/hours/views.py

```diff
@@ -943,6 +944,23 @@ def get_queryset(self, queryset=None):
         return sorted_fiscal_years
 
 
+    def get_context_data(self, **kwargs):
+        # Call the base implementation first to get a context
+        context = super().get_context_data(**kwargs)
+        # Get today's date to figure out the fiscal year it is in
+        today = localdate()
+        # The year will be used to determine which year reporting to end with
+        year = ReportingPeriod.get_fiscal_year_from_date(today)
+        context['fiscal_years'] = [
+            {
+                'year': fy,
+                'start_date': ReportingPeriod.get_fiscal_year_start_date(fy),
+                'end_date': ReportingPeriod.get_fiscal_year_end_date(fy)
+            } for fy in range(2017, year + 1)
```

Comment:
This is interesting how you're building out the list of dicts in a list comprehension. I wouldn't have thought of that, but it's very readable this way!